[rospy.client][INFO] 2021-01-16 07:38:16,826: init_node, name[/train_582_1610782696798], pid[582]
[xmlrpc][INFO] 2021-01-16 07:38:16,827: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2021-01-16 07:38:16,827: Started XML-RPC server [http://192.168.0.191:44237/]
[rospy.init][INFO] 2021-01-16 07:38:16,827: ROS Slave URI: [http://192.168.0.191:44237/]
[rospy.impl.masterslave][INFO] 2021-01-16 07:38:16,828: _ready: http://192.168.0.191:44237/
[rospy.registration][INFO] 2021-01-16 07:38:16,829: Registering with master node http://192.168.0.126:11311
[xmlrpc][INFO] 2021-01-16 07:38:16,829: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2021-01-16 07:38:16,928: registered with master
[rospy.rosout][INFO] 2021-01-16 07:38:16,928: initializing /rosout core topic
[rospy.rosout][INFO] 2021-01-16 07:38:16,933: connected to core topic /rosout
[rospy.simtime][INFO] 2021-01-16 07:38:16,934: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2021-01-16 07:38:16,945: topic[/bev/temporal_bev_image] adding connection to [http://192.168.0.126:36903/], count 0
[rospy.internal][INFO] 2021-01-16 07:38:16,957: topic[/odom] adding connection to [http://192.168.0.126:45431/], count 0
[rospy.internal][INFO] 2021-01-16 07:38:16,959: topic[/is_start_episode] adding connection to [http://192.168.0.126:34007/], count 0
[rospy.internal][INFO] 2021-01-16 07:38:16,963: topic[/scan] adding connection to [http://192.168.0.126:36669/], count 0
[rospy.internal][INFO] 2021-01-16 07:38:16,963: topic[/scan] adding connection to [http://192.168.0.126:45431/], count 0
[rospy.internal][INFO] 2021-01-16 07:38:16,965: topic[/start_goal_points] adding connection to [http://192.168.0.126:40949/], count 0
[rospy.internal][INFO] 2021-01-16 07:38:17,089: topic[/cmd_vel] adding connection to [/gazebo], count 0
[rospy.internal][INFO] 2021-01-16 07:38:17,090: topic[/episode_watcher/is_finish_episode] adding connection to [/episode_manager], count 0
[rospy.internal][INFO] 2021-01-16 07:38:17,187: topic[/rosout] adding connection to [/rosout], count 0
[rospy.core][INFO] 2021-01-16 07:38:19,205: signal_shutdown [atexit]
[rospy.internal][INFO] 2021-01-16 07:38:19,208: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2021-01-16 07:38:19,208: topic[/cmd_vel] removing connection to /gazebo
[rospy.internal][INFO] 2021-01-16 07:38:19,209: topic[/episode_watcher/is_finish_episode] removing connection to /episode_manager
[rospy.internal][INFO] 2021-01-16 07:38:19,209: topic[/bev/temporal_bev_image] removing connection to http://192.168.0.126:36903/
[rospy.internal][INFO] 2021-01-16 07:38:19,209: topic[/odom] removing connection to http://192.168.0.126:45431/
[rospy.internal][INFO] 2021-01-16 07:38:19,209: topic[/start_goal_points] removing connection to http://192.168.0.126:40949/
[rospy.internal][INFO] 2021-01-16 07:38:19,210: topic[/scan] removing connection to http://192.168.0.126:36669/
[rospy.internal][INFO] 2021-01-16 07:38:19,210: topic[/scan] removing connection to http://192.168.0.126:45431/
[rospy.internal][INFO] 2021-01-16 07:38:19,211: topic[/is_start_episode] removing connection to http://192.168.0.126:34007/
[rospy.impl.masterslave][INFO] 2021-01-16 07:38:19,211: atexit
[rospy.internal][WARNING] 2021-01-16 07:38:19,212: Unknown error initiating TCP/IP socket to 192.168.0.126:43067 (http://192.168.0.126:40949/): Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

[rospy.internal][WARNING] 2021-01-16 07:38:19,212: Unknown error initiating TCP/IP socket to 192.168.0.126:57109 (http://192.168.0.126:34007/): Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

